OmniBLE/PumpManager/PodCommsSession.swift

```diff
@@ -48,7 +48,7 @@ extension PodCommsError: LocalizedError {
         case .invalidData:
             return nil
         case .noResponse:
-            return LocalizedString("No response from pod", comment: "Error message shown when no response from pod was received")
+            return LocalizedString("No response from pod; check Bluetooth", comment: "Error message shown when no response from DASH pod was received")
```

Comment:
Suggestions for recovery should go into the `recoverySuggestion` method of `PodCommsError`. And "check bluetooth" could probably be improved. If their bluetooth is turned off or not working, they won't get this  `noResponse`, and I don't think people would know what to check other than that if they see advice of "check bluetooth".  This error is likely to happen if somehow we were maintaining bluetooth connectivity, but dropped a packet.  I think the advice for this should probably be to make sure the phone is close to the pod.

Reply:
If you just remove this for now, I can take the other change, of just changing the button color.

Reply:
done